.github/workflows/publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  push:
+    branches: # Snapshots
+      - main
+    tags: ["*"] # Releases
+
+jobs:
+  publish-artifacts:
+    name: Publish / Artifacts
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Set up JDK 8
```

Comment:
Yep, let's never forget to release with jdk8. 😆

Reply:
Yes, you know that better than me 😂